pkg/controllers/cloud/networkpolicy_controller.go

```diff
@@ -589,8 +595,9 @@ func (r *NetworkPolicyReconciler) Start(stop context.Context) error {
 		case <-ticker.C:
 			r.backgroupProcess()
 			r.retryQueue.CheckToRun()
-			if time.Now().Unix()&cloudSyncInterval == 0 {
+			if time.Now().Unix()-previousTime >= *cloudSyncInterval {
```

Comment:
this is not logically correct.. you need to reset previousTime whenever cloudSyncInterval is updated with configmap update..

Reply:
Solved